tower/src/balance/p2c/service.rs

```diff
@@ -68,10 +59,10 @@ where
     }
 }
 
-#[pin_project]
 /// A Future that becomes satisfied when an `S`-typed service is ready.
 ///
-/// May fail due to cancelation, i.e. if the service is removed from discovery.
+/// May fail due to cancelation, i.e., if the service is removed from discovery.
```

Comment:
maybe worth linking to something in the `discover` module that's relevant to cancelation?